Code/GraphMol/Depictor/EmbeddedFrag.cpp

```diff
@@ -346,6 +346,76 @@ void EmbeddedFrag::setupAttachmentPoints() {
   }
 }
 
+// check if the stereochemistry of the template matches the stereochemistry of
+// the molecule
+static bool checkStereoChemistry(const RDKit::ROMol &mol,
+                                 const RDKit::ROMol &template_mol,
+                                 RDKit::MatchVectType match) {
+  std::cerr << "checkStereoChemistry" << std::endl;
```

Comment:
please remove all debugging output like this

Reply:
gone